src/web-auth/index.js

```diff
@@ -381,15 +381,15 @@ WebAuth.prototype.signupAndAuthorize = function (options, cb) {
 };
 
 /**
- * Logs in the user with username and password using the cross origin authentication flow. You can use `username` or `email` as the actual username.
+ * Logs in the user with username and password using the cross origin authentication flow. You can use either `username` or `email` to identify the user.
  *
  * @method login
  * @param {Object} options options used in the {@link authorize} call after the login_ticket is acquired
- * @param {String} options.username username
- * @param {String} options.email email
- * @param {String} options.password user password
- * @param {String} options.realm realm
- * @param {Function} cb callback function called only when an authorization error occurs. Has the error as the only parameter.
+ * @param {String} [options.username] Username
+ * @param {String} [options.email] Email
+ * @param {String} options.password Password
+ * @param {String} [options.realm] Use the realm paramater to specify a connection. Ex: 'Username-Password-Authentication'
```

Comment:
`realm used to authenticate the user, it can be a realm name or a database connection name`